imports/ui/components/journalPage/Page.jsx

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import Entry from './Entry';
+import ButtonNewPage from './ButtonNewPage';
+
+class Page extends Component {
+    
+    render() {
+        if (this.props.page == "right") {
+            return (
+                <div className="grid-item right-page">
+                    <Entry header="Jun 8 | Sat"/>
```

Comment:
We need to refactor these dummy data out to a separate mock data file, so that it's easier for us to integrate with the backend later on.